app/src/main/java/org/sil/gatherwords/room/Session.java

```diff
@@ -17,7 +17,7 @@
 public class Session {
 	// Define the primary key
 	@PrimaryKey(autoGenerate = true)
-	public int id = 0;
+	public Long id;
```

Comment:
What is the purpose of this change?  If you want a larger key space, why not use the primitive `long`?  Does this schema change require changing the db version?

Reply:
`int` wasn't an option for returning the id's after an insert